docs/migration.md

```diff
@@ -0,0 +1,287 @@
+Ember Mirage does not provide a lot of the features that ember-cli-mirage does. It is many of those fetaures
+that is causing the ember-data issues as well as preventing support for embroider. 
+
+Ember mirage does not support the following. 
+    auto discover of models
+    auto discover of serializers
+    auto load of mirage files (models/serializers/factories/etc)
+    auto discover of the server config
+    auto start of mirage in development
+
+This guides goes through what you need to do to migrate from ember-cli-mirage and how to accomplish the things needed
+to use mirageJS
+
+ECM also contains code to exclude the mirage files during certain builds, for example production. This addon 
+does not include that feature and instead relies on embroider. If you are not on embroider yet, you may
+desire to wait until an non-embroider solution is completed. 
+
+Steps 1 - 5 should be completed in ember-cli-mirage prior to converting to ember-mirage, Cosider this a form 
+of removing deprecations so that you can move to the next version (this addon).
+
+1) Remove the use of autoDiscoverModels.
+
+    Since ECM auto discovers your models, it creates a mirage model even if you dont need it. Sicne we dont know what 
+models you actually use, you will need to create a mirage model for every model in ember data. Using the blueprints
+do `ember g mirage-model model-name` for each model you have. This will create a file in your midage/models directory. 
+Edit each model and add the belongsTo and hasMany. Mirage does not require you to define the attributes. 
+
+
+```js
+//app/models/account.js
+import Model, { belongsTo, hasMany, attr } from '@ember-data/model';
+
+export default class AccountModel extends Model {
+    @attr
+    name;
+
+    @belongsTo('accountType', { async: true, inverse: null }),
+    accountType;
+    
+    @hasMany('product', { async: true, inverse: null }),
+    products;
+}
+```
+would look like this in MirageJS. See the mirage guides if you have any questions on 
+defining models [MirageJS Models](https://miragejs.com/docs/main-concepts/models/)
+
+```js  
+//mirage/models/accoutn.js
+import { Model, belongsTo, hasMany } from 'miragejs';
+
+export default Model.extend({
+    accountType: belongsTo(),
+    products: hasMany(),
+});
+```
+
+Your make server function previously looked like this
+```js
+export default function (config) {
+  let finalConfig = {
+    ...config,
+    models: { ...discoverEmberDataModels(), ...config.models },
+    routes,
+  };
+
+  return createServer(finalConfig);
+}
+```
+It will not look like this
```

Comment:
```suggestion
It will now look like this
```

Reply:
fixed. Just now saw that "commit suggestion" is an option. Will have to try that sometime